packages/web-config-server/src/export/buildExportUrl.js

```diff
@@ -6,4 +6,4 @@
 import { stringifyQuery } from '@tupaia/utils';
 
 export const buildExportUrl = (req, resource, queryParams) =>
-  stringifyQuery(`//${req.headers.host}${req.baseUrl}`, `export/${resource}`, queryParams);
+  stringifyQuery('', `export/${resource}`, queryParams);
```

Comment:
So URL is now just `export/${resource}`? I suppose this makes the `req` argument redundant but I guess we don't wanna remove it now and increase the scope of the refactor?

As long as it's noted that we need to clean this up 👍

Reply:
Cleaned up, good spotting